### ISSUE TITLE
Cannot run batch processing of multiple files

### ISSUE THREAD

**Issue by sarahapark** [MouseLand/suite2p#294]
Hello!
I updated suite2p on today and now cannot batch process multiple files. Is this a problem other people have run into? What must I do to fix it?

Thank you!
Sarah

**Comment by DoriMG** [comment]
Hi!
I also have a problem with batches. I just updated suite2P this morning, and now when I put in a batch it only runs the first file, and then finishes. Is this the same issue you're having, Sarah?
Cheers, Dori

**Comment by carsen-stringer** [comment]
are you using the latest github version? are there any errors on the command line?

**Comment by DoriMG** [comment]
I tried to replicate the error this morning, and actually couldn't, it's working fine now for me. I didn't update anything in the meantime, so not sure what was going on. 

I do get an error when adding files to the batch, but this seems to just be gui error that doesn't have any consequences as far as I could tell. This was the error: 

`===== 2020.03.05 12:03:54 =====
Traceback (most recent call last):
  File "C:\Users\Dori\Anaconda3\lib\site-packages\suite2p\gui\rungui.py", line 357, in list_ops
    self.h5text.setText('')
AttributeError: 'RunWindow' object has no attribute 'h5text'`

**Comment by sarahapark** [comment]
Hi Dori,
Yes, I am having the same problem as you describe. The "save settings and add more (batch)" doesn't work after the first file. It is not adding files to the batch, as Carsen said. I updated suite2p the day I posted (4 days ago), so I assume it is the latest github version? I updated on Anaconda by typing "pip install suite2p --upgrade".

**Comment by sarahapark** [comment]
I am running an un-updated suite2p on another computer, and the batch processing is working fine on that computer.

**Comment by carsen-stringer** [comment]
I think all these problems are resolved in the github version, which you install with the following instructions: https://github.com/MouseLand/suite2p#installation-of-github-version-for-developers-and-super-users-

(not pip install suite2p --upgrade)

**Comment by sarahapark** [comment]
Hi Carsen,
I tried following the instructions
<https://github.com/MouseLand/suite2p#installation-of-github-version-for-developers-and-super-users->to
install the github version of suite2p, but I ran into this problem (see
image below). How can I solve this?
In which directory should I clone the environment.yml (or '.txt'?)
repository? Sorry. I know you closed this request, but I need a bit more
help to proceed!

[image: image.png]
Thank you!
Sarah
